bindings/cpp/include/wtf/macros.h

```diff
@@ -44,6 +45,19 @@
 #define WTF_THREAD_ENABLE(name) \
   WTF_THREAD_ENABLE_IF(kWtfEnabledForNamespace, name)
 
+// Enables WTF tracing for any thread based on whether the current namespace
+// is enabled. Thread name will be auto-generated.
+#define WTF_ALL_THREADS_ENABLE() \
```

Comment:
Hi Stella, thank you for the review! I am short on free time this week, so will be reassigning the task to you. Thanks for help!